pandas/core/frame.py

```diff
@@ -7909,11 +7909,17 @@ def append(
             to_concat = [self, *other]
         else:
             to_concat = [self, other]
-        return concat(
-            to_concat,
-            ignore_index=ignore_index,
-            verify_integrity=verify_integrity,
-            sort=sort,
+        return (
```

Comment:
Maybe leave this one off for now. We haven't really defined the behavior for multiple pandas objects yet. I think it'll be something like `result.__finalize((self, other), method="append")`, but that'll require some discussion.